src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -120,6 +121,10 @@ public SliceFromConfig(final RepoConfig config, final Vertx vertx,
                         "gem",
                         config -> CompletableFuture.completedStage(new GemSlice(storage, vertx.fileSystem()))
                     ),
+                    new MapEntry<>(
+                        "helm",
+                        config -> CompletableFuture.completedStage(new HelmSlice(storage))
```

Comment:
@Sammers21 do you need full path URIs for Helm? If not, please wrap it with `TrimPathSlice` as some other endpoints

Reply:
I need the full path